src/Tokenizer/TokensAnalyzer.php

```diff
@@ -63,6 +63,12 @@ public function getClassyElements()
                 continue;
             }
 
+            if ($token->isClassy()) { // anonymous class in class
+                $classCurlyBraces = 0;
```

Comment:
if you have anonymous class in regular class, won't this line break detection of end of regular class ?